jetbrains-core/src/software/aws/toolkits/jetbrains/utils/NotificationUtils.kt

```diff
@@ -80,7 +81,7 @@ fun notifyNoActiveCredentialsError(
         title = title,
         content = content,
         project = project,
-        action = SettingsSelectorAction(showRegions = false)
+        action = SettingsSelectorAction(ChangeAccountSettingsMode.CREDENTIALS)
```

Comment:
Can't comment on the function...

Where is this used? Will this lead to confusion if its conflated with no region?

Reply:
this is specifically `nonotifyNoActiveCredentialsError`

Reply:
but looking at where it's used, it could be either - will fix###

Reply:
actually - i'm going to do this separately - because it relies on logic that is already in the next PR